docs/API.md

```diff
@@ -2230,7 +2228,10 @@ HTTP/1.1 200 OK
 Content-Type: application/json
 Content-Length: 2
 
-{}
+{
+    "recoveryKeyId": "6aa248931704886f54ac64b81b111bc0",
+    "recoveryData": "eyJlbmMiOiJBMjU2R0NNIiwiYWxnIjoiZGlyIiwia2lkIjoiNmFhMjQ4OTMxNzA0ODg2ZjU0YWM2NGI4MWIxMTFiYzAifQ"
+}
```

Comment:
It seems weird to have an API named "/recoveryKeys" plural that returns only a single item.  I worry that this could be a source of potential future confusion and footguns.  Can we either change the API to be "recoveryKey" singular, or change this to return return a list of keys that might contain more than one in the future?

Reply:
I think I will go with renaming it to be singular. This makes the most sense to me since we don't have a timeline for supporting multiple keys yet.